src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
 }
+
+@import './variables';
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to put all `@import` statements at the top of the file, above all other rules to improve file structure and clarity.